.env

```diff
@@ -12,4 +12,4 @@ DB_PASSWORD=postgres
 EXPORT_BUCKET=mbta-ctd-dataplatform-dev-springboard
 ARCHIVE_BUCKET=mbta-ctd-dataplatform-dev-archive
 ERROR_BUCKET=mbta-ctd-dataplatform-dev-error
-IMPORT_BUCKET=mbta-ctd-dataplatform-dev-incoming
+INGEST_BUCKET=mbta-ctd-dataplatform-dev-incoming
```

Comment:
keep us consistent with whats going on in the lambda side of things.

we should eventually standardize these.